sdk/servicebus/Azure.Messaging.ServiceBus/samples/TopicFilters/TopicFilters.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net5.0</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
```

Comment:
Should we consider hooking into the central package versions when building from within the SDK repo?

Reply:
Ah yes, forgot about that.

Reply:
Created https://github.com/Azure/azure-sdk-for-net/issues/20842 to track this.